Navtor_intern_task/FleetManagementAppTest/ShipTest.cs

```diff
@@ -0,0 +1,35 @@
+using FleetManagementApp;
+
+namespace FleetManagementAppTest;
+
+public class ShipTest
+{
+    [Fact]
+    public void Constructor_InvalidId_ThrowsArgumentException()
+    {
+        // Arrange
+        string invalidId = "IMO8593506";
+
+        // Act & Assert
+        Assert.Throws<ArgumentException>(() => 
+            new ContainerShip(
```

Comment:
Lots of copy paste to create ship instances in all of the tests, maybe extracting it out as a method (with optional and default params) or a builder pattern would be nice?